Mathlib/Analysis/SpecialFunctions/Arcosh.lean

```diff
@@ -115,4 +127,63 @@ def coshPartialEquiv : PartialEquiv ℝ ℝ where
   left_inv' _ hr := arcosh_cosh hr
   right_inv' _ hr := cosh_arcosh hr
 
+theorem continuousOn_arcosh : ContinuousOn arcosh (Ici 1) :=
+  have {x : ℝ} (hx : x ∈ Ici 1) : 0 < x + √(x ^ 2 - 1) :=
+    add_pos_of_pos_of_nonneg (show 0 < x by grind) (sqrt_nonneg _)
+  continuousOn_log.comp (Continuous.continuousOn (by continuity)) (fun _ _ => by grind)
```

Comment:
```suggestion
  continuousOn_log.comp (Continuous.continuousOn (by continuity)) (by grind [MapsTo])
```
This lets `grind` unfold the definition, instead of invoking it only after the `intro` (or `fun`) unfolds it